rpmlint/configdefaults.toml

```diff
@@ -331,14 +327,6 @@ binaries = false
 # dangling symlink is shipped
 [DanglingSymlinkExceptions]
 
-[FileDigestLocation]
-
-[FileDigestGroup]
-
-[DeviceFilesWhitelist]
-
-[WorldWritableWhitelist]
-
```

Comment:
Do we need to move these empty config entries to `opensuse.toml`? Otherwise the checks might fail when there are no explicit whitelistings like with devicefileswhitelist.

Reply:
Yes, you are right, I missed that. I will fix it